lang/cs/Org.Apache.REEF.IO.Tests/TestAzureDataLakeFileSystem.cs

```diff
@@ -210,13 +210,13 @@ public void TestGetFileStatusThrowsException()
 
         private sealed class TestContext
         {
-            public readonly string AdlAccountName = "adlAccount";
+            public readonly string AdlAccountFQDN = "adlAccount.azuredatalakestore.net";
```

Comment:
"AdlsAccountFqdn". You're missing the "s" in "adls". Also, the naming convention for acronyms is code is still camel case (see https://msdn.microsoft.com/en-us/library/ms229043(v=vs.100).aspx)

Reply:
There is adls, adl, and adlfs scattered throughout, I'll change them all the adls to be consistent.